lib/main.py

```diff
@@ -45,10 +47,14 @@ def signout():
 
 def main():
     global BACKGROUND
-    with util.Cron(1):
-        BACKGROUND = background.BackgroundWindow.create(function=_main)
-        BACKGROUND.modal()
-        del BACKGROUND
+    util.setGlobalProperty('running', '1')
+    try:
+        with util.Cron(1):
+            BACKGROUND = background.BackgroundWindow.create(function=_main)
+            BACKGROUND.modal()
+            del BACKGROUND
+    finally:
+        util.setGlobalProperty('running', '0')
```

Comment:
I tend to like these properties to be cleared (unless it really is a multi-state property) so we can use the simpler `String.IsEmpty()` to check them.

Reply:
OK. I was just following your snippet in chat :)

Reply:
Although I wouldn't be shocked if I accidentally did that in my snippet, I don't see it in my snippet :smile: